lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_bump_version_final_release.rb

```diff
@@ -8,11 +8,20 @@ def self.run(params)
         require_relative '../../helper/android/android_version_helper'
 
         Fastlane::Helper::GitHelper.ensure_on_branch!('release')
-        create_config()
-        show_config()
+        app = ENV['PROJECT_NAME'].nil? ? params[:app] : ENV['PROJECT_NAME']
 
-        UI.message 'Updating gradle.properties...'
-        Fastlane::Helper::Android::VersionHelper.update_versions(@final_version, @current_version_alpha)
+        current_version = Fastlane::Helper::Android::VersionHelper.get_release_version(app)
+        current_version_alpha = Fastlane::Helper::Android::VersionHelper.get_alpha_version(app)
+        final_version = Fastlane::Helper::Android::VersionHelper.calc_final_release_version(current_version, current_version_alpha)
+
+        vname = Fastlane::Helper::Android::VersionHelper::VERSION_NAME
+        vcode = Fastlane::Helper::Android::VersionHelper::VERSION_CODE
+        UI.message("Current version[#{app}]: #{current_version[vname]}(#{current_version[vcode]})")
+        UI.message("Current alpha version[#{app}]: #{current_version_alpha[vname]}(#{current_version_alpha[vcode]})") unless current_version_alpha.nil?
+        UI.message("New release version[#{app}]: #{final_version[vname]}(#{final_version[vcode]})")
+
+        UI.message 'Updating version.properties...'
```

Comment:
To be fair it only updates `version.properties` if there is one, otherwise it updates `build.gradle` 😛

Reply:
😅